sycl/doc/extensions/experimental/sycl_ext_oneapi_graph.asciidoc

```diff
@@ -516,14 +516,16 @@ update support:
 namespace sycl {
 enum class aspect {
   ...
-  sycl_ext_oneapi_graph_update
+  sycl_ext_oneapi_limited_graph
```

Comment:
```suggestion
  ext_oneapi_limited_graph
```

Extended enumerators should start with the prefix "ext_\<vendorstring\>", not "sycl_ext_\<vendorstring\>".  See [section 6.3.2](https://registry.khronos.org/SYCL/specs/sycl-2020/html/sycl-2020.html#_names_for_extensions_to_existing_classes_or_enumerations) of the core SYCL specification.

Reply:
This is fixed now, I had referenced another extension document which makes this same mistake.

Reply:
Which one?